lib/franklin_web/live/admin/post_editor_live.ex

```diff
@@ -20,14 +22,41 @@ defmodule FranklinWeb.Admin.PostEditorLive do
 
     case apply_action(PostForm.changeset(%PostForm{}, form_params), :validate) do
       {:error, changeset} ->
+        IO.inspect(changeset, label: "changeset")
+
         socket
         |> assign(changeset: changeset)
         |> noreply()
 
-      {:ok, _} ->
+      {:ok, _changeset} ->
         # Make the `CreatePost` command, dispatch it, then wait for an event to signal it is projected, then do a redirect.
+        # I don't think we want to make a command here.
+        # Commands should be a hidden implimentation of the core
+        # We can't send the core this changeset though since that is a UI detail
+        uuid = Ecto.UUID.generate()
+        title = nil
+
+        # for now we'll do a raw datetime value and later we will parse the string into a datetime
```

Comment:
As this documents, these attrs are handcrafted for experimentation. See issue #4 for work to be done to fix this page.